WCON_format.md

```diff
@@ -129,23 +131,29 @@ A WCON file with arrayed t and corresponding nested arrays for `x` and `y`.
 }
 ```
 
-Note that even if some xy-coordinate arrays are nested, it is still possible to repeat an `id` in the same file if the same worm has more tracks later.  Allowing repeats in `id` is convenient for multi-worm tracking data as tracks can be joined simply by updating `id` without having to copy track data.
```

Comment:
Did you remove this section because you feel this section suffices in establishing that we can repeat `id`s in multiple tracks in the same file?  Just making sure there is no change of policy here.

> Note: A data array may contain the same ID multiple times with different timepoints.  It should _not_ contain the same ID with the same timepoint multiple times.  Allowing repeats enables easy merging and splitting of data across several files or chunks of analysis.